ci_framework/roles/polarion/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_polarion"
+
+cifmw_polarion_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
+cifmw_polarion_jump_repo_dir: "{{ cifmw_polarion_basedir }}/jump"
+cifmw_polarion_jump_result_dir: "{{ cifmw_polarion_basedir }}/artifacts/tempest/"
```

Comment:
@arxcruz care to validate this location, since you're working on #216 ?

Reply:
Yes please, i think in practice we will need to always re-point it to basedir/tests/tempest/ (unless later we will have need for consuming results from url which would need different handling completely).

Reply:
Changed result_dir to tests/tempest/ according to #216